src/main/java/generics/Box.java

```diff
@@ -0,0 +1,75 @@
+package generics;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+public class Box {
```

Comment:
```suggestion
public class Box<E extends Fruit> {
```
Десь так має бути, а далі потрібно використовувати Е усюди.